app/models/UserAnswers.scala

```diff
@@ -27,6 +28,7 @@ final case class UserAnswers(
                               id: String,
                               data: JsObject = Json.obj(),
                               smallProducerList: List[SmallProducer] = List.empty,
+                              packagingSiteList: List[Site] = List.empty,
```

Comment:
packagingSiteList with type List[Site] isn't very supportive for the update and delete elements to come in later stories. A better approach would be to use a Map[String, Site]. That way you can have a key for the Site being updated or removed. Unlike the smallProducerList which has a identifier within the SmallProducer class, the Site model doesn't. I started doing this with Jake so he could show you.

Reply:
done